### PR TITLE
Harmonize Font Sizes

### PR DESCRIPTION
**Pull request by keckelt**
Closes https://github.com/Caleydo/cohort/issues/670
part of https://github.com/Caleydo/cohort/issues/666

* Removed the various font-size differences in the search bar dropdown.
  The main attributes and sub-list entries are all the same size now.
* Converted `px` sizes to `em`, unless they related to line height or similar
* Made Welcome Text larger
* Made task chooser text larger

Coral App:
font size 1

Operation Selection Pane:
image

### CODE REVIEW

File: coral/db.py [hunk 1]
```diff
@@ -28,10 +28,8 @@
   for col in columns[table]:
       db_builder.column(col[0], type=col[1])
 
-  # assign_ids ... the tdp server should automatically manage and assign unique integer ids based on the 'id' column
   # call(inject_where) ... utility to inject a where clause that is used for dynamic filtering
-  views[table] = db_builder.assign_ids() \
```
Comment (keckelt) [inline coral/db.py:33]: Branch is based on #20 (unintentionally)

File: src/Taskview/SearchBar.ts [hunk 2]
```diff
@@ -845,8 +845,6 @@ export class SearchBar {
       log.debug(`create detail for ${data.optionId}`);
       if ((data as ISpecialOption).optionData && (data as ISpecialOption).optionData.spAttribute) {
         detail = this._createSpecialDetail(data as ISpecialOption);
-      } else {
-        detail = this._createDBColumnDetail(data as IServerColumnOption);
```
Comment (keckelt) [inline src/Taskview/SearchBar.ts:849]: No details for db columns, see https://github.com/Caleydo/cohort/issues/666

File: src/app.ts [hunk 3]
```diff
@@ -211,7 +211,7 @@ export class CohortApp {
 
     this.restartSession = btnGrp.append('button')
       .attr('type', 'button')
-      .attr('class', 'btn btn-coral')
```
Comment (keckelt) [inline src/app.ts:214]: Button was larger than the dataset buttons

**Review by PatrickAdelberger — APPROVED** [review]
okay :+1:

**Pull request merged**